src/genie/libs/parser/iosxe/show_vrf.py

```diff
@@ -62,7 +62,7 @@ def cli(self, vrf='', output=None):
         # vpn4                             100:2                 ipv4,ipv6
         # rb-bcn-lab                       10.116.83.34:1        ipv4,ipv6   Lo9
         # test                             10.116.83.34:100      ipv4,ipv6   Lo100
-        p1 = re.compile(r'^(?P<vrf>[\w\d\-\.]+)\s+(?P<rd>\<not +set\>|[\.\d\:]+)\s+'
+        p1 = re.compile(r'^(?P<vrf>[\w\d\-\.\:]+)\s+(?P<rd>\<not +set\>|[\.\d\:]+)\s+'
```

Comment:
Please add an example here with the ```:```

Reply:
What kind of example are you looking for?

Reply:
You mean an example of what it wouldn't parse before, but _would_ parse with my code changes?

```
ip vrf CUST:A
 rd 9999:1000
```

Reply:
what @omehrabi meant was, like line 62-64. please add your output line which needs this change. so that we can evaluate the regex later with these example output lines.

Reply:
Added them